internal/home/upgrade.go

```diff
@@ -240,8 +240,9 @@ func upgradeSchema3to4(diskConf yobj) error {
 
 // Replace "auth_name", "auth_pass" string settings with an array:
 // users:
-// - name: "..."
-//   password: "..."
+//   - name: "..."
+//     password: "..."
```

Comment:
Just to be clear, these format changes have been made by the Go 1.19 comment formatter as opposed to manual reformatting, right?

Reply:
As sorry I missed this :-(
Yes, it is the Go 1.10 comment formatter :-/.
I can remove this if you prefer?

Reply:
No, you don't need to, I'm just making sure.